glib/src/win32.rs

```diff
@@ -91,6 +90,9 @@ pub fn package_installation_directory_of_module(
         )
     } {
         Some(pb) => Ok(pb),
-        None => Err(unsafe { GetLastError() }.0),
+        // The `raw_os_error` function is guaranteed to always return Some(..)
+        // because `last_os_error` constructs `Error` instances from raw OS
+        // error values
+        None => Err(std::io::Error::last_os_error().raw_os_error().unwrap()),
```

Comment:
Why not return a `std::io::Error` here?

Reply:
I did that because I thought it's not really intuitive to return an I/O error when calling this function. But thinking again now it makes perfect sense since the function is expected to return a path to a file, not to mention that a random `i32` is not that intuitive either to begin with. Definitely an oops on my end.